src/openrct2/interface/InteractiveConsole.cpp

```diff
@@ -1316,6 +1317,19 @@ static int32_t cc_for_date([[maybe_unused]] InteractiveConsole& console, [[maybe
     return 1;
 }
 
+static int32_t cc_load_park([[maybe_unused]] InteractiveConsole& console, [[maybe_unused]] const arguments_t& argv)
+{
+    if (argv.size() > 0)
+    {
+        char savePath[MAX_PATH];
+        platform_get_user_directory(savePath, "save", sizeof(savePath));
+        safe_strcat_path(savePath, argv[0].c_str(), sizeof(savePath));
+        path_append_extension(savePath, ".sv6", sizeof(savePath));
```

Comment:
It should allow user to include `.sv6` in the name. Personally I would even expect a filename here, without altering it. I guess it should also allow for `.sc6`. I recently was told there's `String::EndsWith` function that can be used for that.

Reply:
Gonna add that soon and will do another PR.

Reply:
This PR won't be merged until after the v0.2.6 release, so please amend this PR instead.